include/bluetooth/mcc.h

```diff
@@ -460,182 +566,361 @@ struct bt_mcc_cb {
 
 /**** Functions ***************************************************************/
 
-/** @brief Initialize Media Control Client
+/**
+ * brief Initialize Media Control Client
  *
  * @param cb    Callbacks to be used
+ *
+ * @return 0 if success, errno on failure.
  */
 int bt_mcc_init(struct bt_mcc_cb *cb);
 
 
-/** @brief Discover Media Control Service
+/**
+ * brief Discover Media Control Service
  *
- * Discover Media Control Service on the server given by the connection
+ * Discover Media Control Service (MCS) on the server given by the connection
  * Optionally subscribe to notifications.
  *
- * Shall be called once after initialization and before using other
- * media control client functionality.
+ * Shall be called once, after media control client initialization and before
+ * using other media control client functionality.
  *
- * @param conn        The connection on which to do MCS discovery
+ * @param conn  Connection to the peer device
  * @param subscribe   Whether to subscribe to notifications
+ *
+ * @return 0 if success, errno on failure.
  */
 int bt_mcc_discover_mcs(struct bt_conn *conn, bool subscribe);
 
-/** @brief Read Media Player Name */
+/**
+ * brief Read Media Player Name
+ *
+ * @param conn  Connection to the peer device
```

Comment:
Huh, I guess the MCC still only supports a single MCS instance, right? 

I swear I had thought it supported multiple. Do we have any plans on adding multiple instance support?

Reply:
At least for the client, that might make sense.  Propose to keep as is for now.